### DISCUSSION TITLE
Pre requisites for java backend repo

### DISCUSSION THREAD

**Discussion by hvt0707** [scaleracademy/twitter-backend-java#22] [Q&A] [answered]
Can anyone guide me about the prereqs for the java backend repo? What software do we require on our local machines? I'm proficient in java but new to open source projects.

**Comment by ohbus** [comment, marked as answer]
@hvt0707 You can start by checking out the [Wiki](https://github.com/scaleracademy/twitter-backend-java/wiki)

You can get started by developing a readme for the documentation from Wiki material and work on #4
